vendor/current-when.js

```diff
@@ -0,0 +1,21 @@
+(function() {
+  var _Ember;
+  if (typeof Ember !== 'undefined') {
+    _Ember = Ember;
+  } else {
+    _Ember = require('ember').default;
+  }
+
+  if (EmberENV && EmberENV._ENABLE_CURRENT_WHEN_SUPPORT !== true) {
+    return false;
+  }
+
+  var linkTo = _Ember.__loader.require('ember-glimmer/components/link-to').default;
```

Comment:
`var LinkTo = Ember.LinkComponent`

Reply:
Interesting. I guess I never realized it was hung off Ember. Should be good to go now